visualization/motion_planning_tasks/src/task_panel.cpp

```diff
@@ -192,7 +192,7 @@ void setExpanded(QTreeView* view, const QModelIndex& index, bool expand, int dep
 	// recursively expand all children
 	if (depth != 0) {
 		for (int row = 0, rows = index.model()->rowCount(index); row < rows; ++row)
-			setExpanded(view, index.child(row, 0), expand, depth - 1);
+			setExpanded(view, index.model()->index(row, 0, index), expand, depth - 1);
```

Comment:
Is this an issue with a new Qt version?

Reply:
Yes, [5.12 deprecated it](https://doc.qt.io/qt-5.12/qmodelindex-obsolete.html)